kratos/includes/io.h

```diff
@@ -117,139 +117,306 @@ class KRATOS_API(KRATOS_CORE) IO
     ///@name Operations
     ///@{
 
+    /**
+     * @brief This method reads one node
+     * @param rThisNode The node to be read
+     */
     virtual bool ReadNode(NodeType& rThisNode)
     {
-        KRATOS_ERROR <<  "Calling base class member. Please check the definition of derived class." << std::endl;
+        KRATOS_ERROR << "Calling base class method (ReadNode). Please check the definition of derived class." << std::endl;
```

Comment:
is this necessary? Afaik KRATOS_ERROR tells you already in which method it threw doesn't it?

Reply:
It is not harmful